fud/fud/exec.py

```diff
@@ -102,7 +47,7 @@ def report_profiling(profiled_stages, durations, is_csv):
     """
     data = Source("", SourceType.String)
 
-    if profiled_stages == []:
+    if not profiled_stages:
```

Comment:
Nit: might be useful to throw a type annotation on this since it isn't obvious this is a list.